examples/svd_power.cu

```diff
@@ -41,19 +41,21 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
 {
   MATX_ENTER_HANDLER();
 
+#if 1
```

Comment:
is this needed?

Reply:
not needed but I use it to toggle between batched and non-batched which has been shown to be useful for debugging/testing.  We can build examples in 20 seconds where as matx_test is 20 minutes.